cmd/cassandra-writer/main.go

```diff
@@ -58,18 +62,24 @@ func main() {
 	}
 	defer csdSession.Close()
 
+	tracer, traceCloser, err := jaegerClient.NewTracer(svcName, cfg.JaegerURL)
+	if err != nil {
+		logger.Fatal(fmt.Sprintf("failed to init Jaeger: %s", err))
+	}
+	defer traceCloser.Close()
+
 	// Create new cassandra-writer repo
-	repo := newService(csdSession, logger)
+	repo := newService(csdSession, logger, tracer)
```

Comment:
instantiate 2 tracers one for the service and the other for pubsub

Reply:
Why do we need two tracers, for other services we're using just one

Reply:
https://github.com/SammyOina/mainflux/blob/56eff66cd6abcef86c9e3f2a0c0f5732947e3699/cmd/http/main.go#L63

https://github.com/SammyOina/mainflux/blob/56eff66cd6abcef86c9e3f2a0c0f5732947e3699/cmd/http/main.go#L76